lib/src/mockserver/server.spec.ts

```diff
@@ -0,0 +1,122 @@
+import nock from "nock";
+import request from "supertest";
+import { ContractDefinition } from "../models/definitions";
+import { TypeKind } from "../models/types";
+import { runMockServer } from "./server";
+
+describe("Server", () => {
+  const proxyBaseUrl = "http://localhost:9988";
```

Comment:
Could you add another test where we don't set a proxy URL? Everything should be mocked in that case.